doc/scripts/GMT_App_M_1d.sh

```diff
@@ -40,28 +41,20 @@ do
 	j=$(expr $i + 1)
 	left=$(sed -n ${j}p tt.lis)
 	right=$(sed -n ${i}p tt.lis)
-	gmt makecpt -H -C$left -T-1/1 > tt.left.cpt
-	gmt makecpt -H -C$left -T-1/1/0.25 > tt.left2.cpt
+	if [ -n "$left" ]; then
+	  gmt makecpt -H -C$left -T-1/1 > tt.left.cpt
+	  gmt makecpt -H -C$left -T-1/1/0.25 > tt.left2.cpt
+  fi
 	gmt makecpt -H -C$right -T-1/1 > tt.right.cpt
 	gmt makecpt -H -C$right -T-1/1/0.25 > tt.right2.cpt
-	gmt colorbar -Dx1.55i/${y}i+w2.70i/0.125i+h+jTC -Ctt.left.cpt -B0
+	if [ -n "$left" ]; then
+	  gmt colorbar -Dx1.55i/${y}i+w2.70i/0.125i+h+jTC -Ctt.left.cpt -B0
+	  gmt colorbar -Dx1.55i/${y2}i+w2.70i/0.125i+h+jTC -Ctt.left2.cpt -Bf0.25
+		echo 1.55 $y ${left} | gmt text -D0/0.05i -F+f9p,Helvetica-Bold+jBC
+	fi
 	gmt colorbar -Dx4.50i/${y}i+w2.70i/0.125i+h+jTC -Ctt.right.cpt -B0
-	gmt colorbar -Dx1.55i/${y2}i+w2.70i/0.125i+h+jTC -Ctt.left2.cpt -Bf0.25
 	gmt colorbar -Dx4.50i/${y2}i+w2.70i/0.125i+h+jTC -Ctt.right2.cpt -Bf0.25
-	gmt text -D0/0.05i -F+f9p,Helvetica-Bold+jBC <<- END
-	1.55 $y ${left}
-	4.50 $y ${right}
-	END
-	if [ $(grep -c HARD_HINGE ${GMT_SHAREDIR}/cpt/${left}.cpt) -eq 1 ]; then # Plot hard hinge symbol for left CPT
-		echo 1.55 $y | gmt plot -St0.2c -Gblack -Wfaint -D0/-0.29i
-	elif [ $(grep -c SOFT_HINGE ${GMT_SHAREDIR}/cpt/${left}.cpt) -eq 1 ]; then # Plot soft hinge symbol for left CPT
-		echo 1.55 $y | gmt plot -St0.2c -Gwhite -Wfaint -D0/-0.29i
-	fi
-	if [ $(grep -c HARD_HINGE ${GMT_SHAREDIR}/cpt/${right}.cpt) -eq 1 ]; then # Plot hard hinge symbol for right CPT
-		echo 4.50 $y | gmt plot -St0.2c -Gblack -Wfaint -D0/-0.29i
-	elif [ $(grep -c SOFT_HINGE ${GMT_SHAREDIR}/cpt/${right}.cpt) -eq 1 ]; then # Plot soft hinge symbol for right CPT
-		echo 4.50 $y | gmt plot -St0.2c -Gwhite -Wfaint -D0/-0.29i
-	fi
+	echo 4.50 $y ${right} | gmt text -D0/0.05i -F+f9p,Helvetica-Bold+jBC -Vi
```

Comment:
Continuing on from https://github.com/GenericMappingTools/gmt/pull/4838#discussion_r579990355, I've partially fixed things so 5 colorbars show up, but not perfect yet. Not nice having a blank space on the top left (because the colorbars are plotted from bottom to top in the script), and the label for the top-right is missing somehow:

![GMT_App_M_1d.png](https://user-images.githubusercontent.com/23487320/108703344-5acaec80-756f-11eb-8c78-8a06c466f5c0.png)

Reply:
L30 (``y0=$(gmt math -Q $n2 $dy MUL 0.5 MUL 0.1 ADD =)`` controls the plot height. Try changing **0.1 ADD** to **0.25 ADD**:

from ``y0=$(gmt math -Q $n2 $dy MUL 0.5 MUL 0.1 ADD =)``
to     ``y0=$(gmt math -Q $n2 $dy MUL 0.5 MUL 0.25 ADD =)``

Reply:
Good suggestion. Committed in d7a084b46d8c49b7390d7229d598357d357a586a and the 'vik0' label now shows up:

![GMT_App_M_1d.png](https://user-images.githubusercontent.com/23487320/108770417-2252ff00-75bf-11eb-85c3-0c0661e4eb67.png)

Still not liking the empty top left corner though, but might be a lot of work to refactor the script to handle this odd case.